src/kagglehub/clients.py

```diff
@@ -50,19 +52,43 @@ def __init__(self):
 
     def get(self, path: str) -> dict:
         url = self._build_url(path)
+        headers = {
```

Comment:
Can you move `f"kagglehub/{kagglehub.__version__}"` to a `KAGGLEHUB_USER_AGENT` var at the top.